core/embed/rust/src/ui/component/text/layout.rs

```diff
@@ -52,6 +52,40 @@ pub struct TextLayout {
     pub continues_from_prev_page: bool,
 }
 
+/// Configuration for chunkifying the text into smaller parts.
+#[derive(Copy, Clone)]
+pub struct Chunks {
+    /// How many characters will be grouped in one chunk.
+    pub chunk_size: usize,
+    /// How big will be the space between chunks (in pixels).
+    pub x_offset: i16,
+    /// Optional characters that are wider and should be accounted for
+    pub wider_chars: Option<&'static str>,
```

Comment:
Kinda hate this tbh (as well as the whole idea of trying to render variable-width font as a fixed-width font). Trying to think how to make this more general. Can we instead remember the width of a normal (or smallest) character and keep track of the actual chunk size when processing chunk?

Reply:
Nor am I happy with this, but this "almost-mono-font" can hardly be fixed to be "completely-mono", because of its small width, as I am describing [here](https://github.com/trezor/trezor-firmware/pull/3237#issuecomment-1715136060)

Not sure I follow your suggestion - can you prototype some code for it?

Merging this does not need to be final, we can improve it in the future